InvenTree/users/admin.py

```diff
@@ -17,9 +17,9 @@
 class ApiTokenAdmin(admin.ModelAdmin):
     """Admin class for the ApiToken model."""
 
-    list_display = ('key', 'user', 'name', 'expiry', 'active')
-    fields = ('user', 'name', 'revoked', 'expiry')
-    readonly_fields = ('key', 'created')
+    list_display = ('token', 'user', 'name', 'expiry', 'active')
+    fields = ('token', 'name', 'revoked', 'expiry')
+    readonly_fields = ('token', 'created')
```

Comment:
Question: Why is that changed? The key is normally used for identification in systems like this

Reply:
The "token" is the redacted version, once it is created the raw value is not shown in the admin interface - I believe this is what you requested?